charts/openshift-metering/templates/presto/presto-worker-config.yaml

```diff
@@ -13,12 +13,29 @@ data:
     node.environment=production
 
   config.properties: |
+    coordinator=false
+    node-scheduler.include-coordinator={{ .Values.presto.spec.presto.config.nodeSchedulerIncludeCoordinator }}
+    discovery-server.enabled=true
+
+    {{- if .Values.presto.spec.presto.config.useTLS }}
```

Comment:
Indent is off

Reply:
It also references an invalid field. I updated both in the most recent commit.